pkg/network/protocols/http/ebpf_main.go

```diff
@@ -161,6 +145,23 @@ func newEBPFProgram(c *config.Config, offsets []manager.ConstantEditor, sockFD *
 		subprograms = append(subprograms, openSSLProg)
 	}
 
+	tailCalls := []manager.TailCallRoute{
+		{
+			ProgArrayName: protocolDispatcherProgramsMap,
+			Key:           uint32(ProtocolHTTP),
+			ProbeIdentificationPair: manager.ProbeIdentificationPair{
+				EBPFFuncName: "socket__http_filter",
+			},
+		},
+		{
+			ProgArrayName: protocolDispatcherClassificationPrograms,
+			Key:           uint32(DispatcherKafkaProg),
+			ProbeIdentificationPair: manager.ProbeIdentificationPair{
+				EBPFFuncName: "socket__protocol_dispatcher_kafka",
+			},
+		},
```

Comment:
should be added only if kafka is enabled

Reply:
Done